.github/ISSUE TEMPLATES/node-related-package-releases.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Node-related package releases
+about: Use this template for tracking package releases that pertain to a specific release of `cardano-node`
+title: "Release: for cardano-node [VERSION]"
```

Comment:
It is not necessarily that always the node release is driving the release, so maybe rewording this a bit would be a good idea.

Maybe title should be about a general release to CHaP and then later having a section that requires to mention if this release is needed for a specific cardano-node version release